Assets/HoloToolkit/Input/Scripts/InputSources/GesturesInput.cs

```diff
@@ -3,7 +3,7 @@
 
 using UnityEngine;
 
-#if UNITY_EDITOR || UNITY_WSA
```

Comment:
Why did all the UNITY_EDITOR directives get removed? It's useful to use these APIs in the editor when using Unity remoting.